cogs5e/sheetManager.py

```diff
@@ -933,7 +929,7 @@ async def beyond(self, ctx, url: str, *, args=""):
         await self.import_sheet(ctx, url, args=args)
 
     @staticmethod
-    async def _load_sheet(ctx, parser, args, loading):
+    async def _load_sheet(ctx, parser, args, loading, version):
```

Comment:
I still think getting the version from `args` rather than passing it through as a function parameter works better here, since you're already parsing the args regardless when you load the character, you might as well pull that out into its own variable and check that when you are setting the version.